src/adapter/src/catalog/open.rs

```diff
@@ -1803,6 +1806,10 @@ impl BuiltinBootstrapClusterSizes {
             Ok(self.system_cluster.clone())
         } else if cluster_name == mz_catalog::builtin::MZ_INTROSPECTION_CLUSTER.name {
             Ok(self.introspection_cluster.clone())
+        } else if cluster_name == mz_catalog::builtin::MZ_PROBE_CLUSTER.name {
+            Ok(self.probe_cluster.clone())
+        } else if cluster_name == mz_catalog::builtin::MZ_SUPPORT_CLUSTER.name {
+            Ok(self.system_cluster.clone())
```

Comment:
I'm not so sure about defaulting this to mz_system.

Reply:
Is it time to bite the bullet and introduce separate, dedicated CLI args for each of the builtin cluster's sizes?